addons/website_form/views/snippets/s_website_form.xml

```diff
@@ -48,21 +48,23 @@
 
         <!-- Add Field Form -->
         <div data-js="AddFieldForm" data-selector=".s_website_form" data-target="form">
-            <we-button class="o_we_text_success"
+            <we-button class="pr-2"
```

Comment:
The spec also says and shows to use the primary color, there is a class just like o_we_text_success for that, something like o_we_text_primary I guess? It should look like in the screenshot.

Also using the pr-X classes for the editor UI is not a good idea... especially since here you use it to compensate a random spacing which is due to the fa-fw class. Remove the fa-fw class and you will see the spacing is automatically correct. However, the + icon feels big and not vertically centered. Maybe the easier thing to do is to totally remove it and use a + text symbol.

Reply:
I can use `o_we_text_brand_primary` but it doesn't look like the screenshot from the spec, and a primary button has also another style than what is on the screenshot, I will clarify with SBU

Reply:
I see this which seems to have the same style? `<we-button class="o_we_bg_brand_primary" data-replace-media="true" data-no-preview="true">Replace</we-button>`

Reply:
Looks like what is expected. Thanks !